File: Sandbox_Topology/PLineTopological.vb
Public Class PLineTopological

    Private _v As List(Of Int32)
    Private _i As Int32     'internal indexing of the lines

    Public Sub New(ByVal V As List(Of Int32), ByVal I As Int32)

        _v = V
        _i = I

    End Sub

    '##### PROPERTIES #####

    Public ReadOnly Property Index As Int32
        Get
            Return _i
        End Get
    End Property

    Public ReadOnly Property PointIndices As List(Of Int32)
        Get
            Return _v
        End Get
    End Property

End Class

File: Sandbox_Topology/PLineTopological.cs
using System.Collections.Generic;

namespace Sandbox
{
    /// <summary>
    /// 
    /// </summary>
    public class PLineTopological
    {

        private List<int> _v;
        private int _i;     // internal indexing of the lines

        /// <summary>
        /// 
        /// </summary>
        /// <param name="V"></param>
        /// <param name="I"></param>
        public PLineTopological(List<int> V, int I)
        {

            _v = V;
            _i = I;

        }

        // ##### PROPERTIES #####

        /// <summary>
        /// 
        /// </summary>
        public int Index
        {
            get
            {
                return _i;
            }
        }

        /// <summary>
        /// 
        /// </summary>
        public List<int> PointIndices
        {
            get
            {
                return _v;
            }
        }

    }
}